### ISSUE TITLE
Unable to import latest version

### ISSUE THREAD

**Issue by kenofori** [annulen/webkit#730]
Hi there,

Sorry for this noobish question

I was able to build QtWebKit 5.212.0 Alpha 2 fine. Then i added Qt += webkit in my .pro. 
When i try to import QtWebKit in my qml based project i only see following versions in intellisense
import QtWebKit 3.0
import QtWebKit 1.0

But not the import QtWebKit 5.212.0 but it throws error.

Please advise

Thanks

**Comment by annulen** [comment]
`import QtWebKit 3.0` is correct, see http://doc.qt.io/archives/qt-5.5/qtwebkit-qmlmodule.html

>/usr/local/lib/QtWebKit

Recommended way is installing QtWebKit into same prefix with Qt, i.e. path where it's `include`, `lib`, `mkspecs` directories are located. This is what build system is trying to do by default. Maybe there is a bug specific to macOS there. That said, you may be fine with your current installation if you set `QMAKEPATH += /usr/local QML2_IMPORT_PATH=/usr/local/qml`

**Comment by annulen** [comment]
Please try building 5.212 branch (http://code.qt.io/cgit/qt/qtwebkit.git/) or snapshot tarball (http://download.qt.io/snapshots/ci/qtwebkit/5.212/latest/src/submodules/), Qt prefix should be used by default instead of `/usr/local`

**Comment by kenofori** [comment]
Thank you for your quick support, should i uninstall my QT (as i chose a custom location earlier) and reinstall in default location, and then install webkit?

**Comment by annulen** [comment]
>should i uninstall my QT (as i chose a custom location earlier) and reinstall in default location, and then install webkit?

No, it wouldn't change anything. But changing `CMAKE_INSTALL_PREFIX` to Qt directory in qtwebkit and reinstall will help. You can use cmake-gui, ccmake, or edit CMakeCache.txt manually. Also, find variable `KDE_INSTALL_USE_QT_SYS_PATHS` there and set it to `ON`.

>Secondly i tried first approach, but same error here is my user/local directory content and my .pro file after changes.

It makes no sense to specify QMAKEPATH and QML2_IMPORT_PATH in .pro file.

QMAKEPATH is an environment variable that needs to be set when you run qmake, so that it knows where to peek additional modules. When .pro file is parsed, it's too late.

QML2_IMPORT_PATH is an environment variable that needs to be set when you run QML application

> im already using 5.212.0 Alpha 2.

It's severely outdated

**Comment by annulen** [comment]
See also https://github.com/annulen/webkit/wiki/Using-QtWebKit-in-your-project

**Comment by kenofori** [comment]
Thank you for bearing with me, so what i did is download the latest repo as u suggested.

Build it using this command.
`./Tools/Scripts/build-webkit --qt --cmakeargs="-Wno-dev -DQt5_DIR=/Applications/QT5/5.10.0/clang_64/lib/cmake/Qt5 -DCMAKE_PREFIX_PATH=/usr/local/opt/jpeg-turbo"`

Then i installed it using
`make -C WebKitBuild/Release install`

I did this in main
```
#include <QtWebKit>
QtWebView::initialize();
```

Now the only issue i get is 
```
/usr/local/lib/QtWebKit.framework/Headers/QtWebKit:4: error: 'QtWebKit/QtWebKitDepends' file not found
#include <QtWebKit/QtWebKitDepends>
```



<img width="433" alt="screen shot 2018-02-13 at 10 14 30 am" src="https://user-images.githubusercontent.com/30837872/36134454-b67facfe-10a6-11e8-8900-6cb639925cdc.png">

**Comment by annulen** [comment]
Not yet. For the time being, you can build documentation yourself, but it doesn't document all new features yet anyway.
